Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [workspace]
+resolver = "2"
```

Comment:
What was the reason for adding this?

Reply:
Without this, you get this warning when building the JS Transformer: 
```
warning: some crates are on edition 2021 which defaults to `resolver = "2"`, but virtual workspaces default to `resolver = "1"`
note: to keep the current resolver, specify `workspace.resolver = "1"` in the workspace root's manifest
note: to use the edition 2021 resolver, specify `workspace.resolver = "2"` in the workspace root's manifest
```

From reading [the docs](https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html), if everything is building with the 2021 resolver then it's all good.